Update implementation section (#47)
greenelab/xswap-manuscript: content/02.body.md

### Implementation and source code

We implemented our modified version of the XSwap algorithm as an open-source Python package.
The package contains submodules for permuting networks, computing the edge prior, and converting networks between adjacency matrix and edge list formats.
Additionally, we include the analytical approximation of the edge prior and functionality to assign unique identifiers to nodes.

> The Python package is available on the Python Packaging Index under the name "xswap".

Reviewer: ```suggestion
The Python package is [available](https://pypi.org/project/xswap/) on the Python Packaging Index under the name "xswap".
```

> The full source code for our method of degree-preserving network permutation has also been made freely available under the BSD 2-Clause License ([https://github.com/hetio/xswap](https://github.com/hetio/xswap)).

Reviewer: ```suggestion
The full source code for our method of degree-preserving network permutation has also been made freely available under the BSD 2-Clause License (<https://github.com/hetio/xswap>).
```

The edge swap mechanism---implemented in C++ for greater speed---uses a bitset to avoid producing edges which violate the conditions for a swap to be considered valid.
While the full bitset implementation is faster for smaller networks, our package uses a compressed bitset [@arxiv:1709.07821] when a network would occupy memory above a user-adjustable threshold.
In addition to the validity conditions already described, our package allows specific edges to be excluded from permutation, and every network permutation returns both a permuted network and summary information about the numbers of swaps attempted, performed, and the reasons why invalid swaps were rejected.

> In addition to the Python package, we are releasing all the code for the analysis and figure generation ([https://github.com/greenelab/xswap-analysis](https://github.com/greenelab/xswap-analysis)).

Reviewer: ```suggestion
In addition to the Python package, all code to generate the analyses and figures is available at <https://github.com/greenelab/xswap-analysis>.
```

> The manuscript was written using the Manubot software [@doi:10.1371/journal.pcbi.1007128 ], and we are releasing the manuscript source as well ([https://github.com/greenelab/xswap-manuscript](https://github.com/greenelab/xswap-manuscript)).

Reviewer: ```suggestion
The manuscript was written using the Manubot software [@doi:10.1371/journal.pcbi.1007128 ], which allows anyone to provide feedback or modifications via the public repository at <https://github.com/greenelab/xswap-manuscript>.
```
